server_ftp/lib/ftp_server.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from ftplib import FTP
+
+from .abstract_ftp_server import AbstractFtpServer
+
+
+class FtpServer(AbstractFtpServer):
```

Comment:
In class names and other places: Use the fully capitalized FTP. Like FTPServer. This is closer to all the underlying code, and therefore easier to remember and maintain.